src/data/auth.ts

```diff
@@ -25,3 +25,16 @@ export const fetchAuthProviders = () =>
   fetch("/auth/providers", {
     credentials: "same-origin",
   });
+
+export const createAuthUser = async (
```

Comment:
Maybe add the word "for" to make it clear what is happening:
```suggestion
export const createAuthForUser = async (
```